template/$PROJECT_NAME$/config/prod.exs

```diff
@@ -72,10 +72,14 @@ config :guardian, Guardian,
 config :<%= @project_name %>, <%= @project_name_camel_case %>.Web.Endpoint,
   secret_key_base: System.get_env("SECRET_KEY_BASE")
 
+<%= if @is_heroku? do %>
```

Comment:
I don't think this entire config should be within the `is_heroku?`. Otherwise if it's false you don't get any of this config at all. How about if `is_heroku?` is true, it uses the config with the `DATABASE_URL` and if its false it uses the `DATABASE_USER` and `DATABASE_PASSWORD`